README.md

```diff
@@ -3,10 +3,9 @@
 [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/facebookresearch/habitat-sim/blob/master/LICENSE)
 [![Anaconda-Server Badge](https://anaconda.org/aihabitat/habitat-sim/badges/version.svg)](https://anaconda.org/aihabitat/habitat-sim)
 [![Anaconda-Server Badge](https://anaconda.org/aihabitat/habitat-sim/badges/platforms.svg)](https://anaconda.org/aihabitat/habitat-sim)
+[![Documentation](https://img.shields.io/badge/docs-automated-green.svg)](https://aihabitat.org/docs/habitat-sim/)
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
-[![GitHub release (latest by date)](https://img.shields.io/github/v/release/facebookresearch/habitat-sim)](https://github.com/facebookresearch/habitat-sim/releases/latest)
```

Comment:
Strongly disagree with as the Anaconda Release version looks entirely different from the semantic versioning we actually using in setup.py and elsewhere.

Reply:
Can you elaborate more, here is the screen showing the versioning, here badge from Anaconda was cached by Github and will update:
<img width="391" alt="Screen Shot 2020-09-18 at 10 45 28 AM" src="https://user-images.githubusercontent.com/877440/93628841-145bdb00-f99c-11ea-818a-50516e015085.png">
How different the versions are?

Reply:
Discussed with Aaron separately, removed the date from release versioning.